common/flights/src/store_client.rs

```diff
@@ -312,4 +316,29 @@ impl StoreClient {
         let vec = serde_json::from_slice(&put_result.app_metadata)?;
         Ok(vec)
     }
+
+    /// Add User
+    pub async fn add_user(
+        &mut self,
+        _username: &str,
+        _password: &str,
+    ) -> anyhow::Result<AddUserActionResult> {
```

Comment:
maybe `common_exception::Result` is better. because we need ErrorCode.

Reply:
Yes,  all methods of store_client lack properly arranged error code, not caller friendly (if the caller needs to conditionally react to the error).